test/models/post_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class PostTest < ActiveSupport::TestCase
```

Comment:
Let's include testing for post's association with comment, especially with regards to the dependent destroy relationship